src/town_gui.cpp

```diff
@@ -37,12 +38,15 @@
 #include "timer/timer.h"
 #include "timer/timer_game_calendar.h"
 #include "timer/timer_window.h"
+#include "hotkeys.h"
 
 #include "widgets/town_widget.h"
 
 #include "table/strings.h"
 
 #include "safeguards.h"
+#include "window_gui.h"
+#include "window_type.h"
```

Comment:
These aren't needed. They are already implicitly included via querystring_gui.h. 
They definitely must not be included after `#include "safeguards.h"`; the existing `#include "zoom_func.h" is also wrong, but that's not something to fix in this PR.